tests/explainers/test_kernel.py

```diff
@@ -331,3 +334,28 @@ def test_kernel_multiclass_multiple_rows():
     explainer = shap.KernelExplainer(lr.predict_proba, X)
     shap_values = explainer(X.iloc[[0, 1], :])
     np.testing.assert_allclose(shap_values.values.sum(1) + explainer.expected_value, pred, atol=1e-04)
+
+
+@pytest.mark.parametrize("nsamples", [3, 5, 10, 100])
+def test_kernel_logits_zeros_ones_probs(nsamples):
+    iris = sklearn.datasets.load_iris(as_frame=True)
```

Comment:
Suggest adding a link to the issue:

```suggestion
    # GH 3912
    iris = sklearn.datasets.load_iris(as_frame=True)
```